### PR TITLE
new function added

### PR DESCRIPTION
**Pull request by hemantsharma2003**
### Summary 

1.  In publishing and graphic design, Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. Lorem ipsum may be used as a placeholder before final copy is availabl.
2. 
1 ddyz8qnOhFeFKY-_c3tleQ

### CODE REVIEW

File: home.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function divide(a,b)
+{
+    return (a/b)
```
Comment (dvikas) [inline home.js:3]: Hi @hemantsharma2003 why we didn't checked for divide by Zero.

File: home.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+function divide(a,b)
```
Comment (dvikas) [inline home.js:1]: Hi, @hemantsharma2003 Please correct the formatting.

File: home.js (hunk 1 above)
Comment (dvikas) [reply home.js:3]: Hi @hemantsharma2003  I think we also ned to check for divide by zero.

File: home.js (hunk 2 above)
Comment (hemantsharma2003) [reply home.js:1]: Hi, @dvikas sure I'll fix it.

Comment (hemantsharma2003) [reply home.js]: Hi @dvikas its done https://github.com/dvikas/git-testing/pull/1/commits/71ff4118b07e176a32bfd4ccf9d04dd2ba8b4f9d

**Comment by hemantsharma2003** [comment]
Hi @dvikas Please check the issue "Divide by 0". https://github.com/dvikas/git-testing/pull/1/commits/1a5c175dd9ea3624364e5c33185ba9fd4cd8e321

**Review by dvikas — APPROVED** [review]

**Pull request merged**